cmd/argocd/commands/app_actions.go

```diff
@@ -190,3 +186,34 @@ func NewApplicationResourceActionsRunCommand(clientOpts *argocdclient.ClientOpti
 	}
 	return command
 }
+
+func getActionableResourcesForApplication(appIf applicationpkg.ApplicationServiceClient, ctx context.Context, appNs *string, appName *string) ([]*v1alpha1.ResourceDiff, error) {
+	resources, err := appIf.ManagedResources(ctx, &applicationpkg.ResourcesQuery{
+		ApplicationName: appName,
+		AppNamespace:    appNs,
+	})
+	if err != nil {
+		return resources.Items, err
```

Comment:
This throws a nil reference when running locally. Can you just return `nil, err`?

Reply:
Same with the two below